Sources/PrismCore/Prism.swift

```diff
@@ -49,12 +51,15 @@ public class Prism {
         }()
 
         // Wait for styleguide IDs we wish to query
-        let (styleguideIDs, styleguideErrors) = getStyleguideIDs(for: owner)
+        let (unfilteredStyleguideIDs, styleguideErrors) = getStyleguideIDs(for: owner)
 
         errors.append(contentsOf: styleguideErrors)
 
         // Get text styles, colors and spacing separately
         // for each styleguide
+
+        let styleguideIDs = unfilteredStyleguideIDs.filter { id in !ignoredStyleGuideIds.contains(id) }
```

Comment:
Change to two Sets and symmetric difference, should be more efficient

Reply:
@freak4pc  When I change this to your suggestion, for some reason this [unit test](https://github.com/GettEngineering/Prism/blob/6ad1d4f414d87a30c78c30184b8e290ea31649bf/Tests/AssetsSpec.swift#L42) fails, do you think it has to do with the incorrect order

this was the code I used `Set(allStyleguideIds).symmetricDifference(Set(ignoredStyleGuides))`

failure:
```
failed - expected to equal <Duplicate colors found: green, mud, purple, tealDuplicate text styles found: Base Heading, Body, HighlightDuplicate spacings found: spacing-m, spacing-xl, spacing-xs, spacing-xxs>, 
got <Duplicate colors found: green, purple, tealDuplicate text styles found: Base Heading, Body, HighlightDuplicate spacings found: spacing-m, spacing-xl, spacing-xs>
```

Reply:
Doesn't seem to be sorting or order related, 
If you break the message you'll notice there's "mud" missing and "spacing-xxs"

```
Duplicate colors found: green, mud, purple, teal
Duplicate text styles found: Base Heading, Body, Highlight
Duplicate spacings found: spacing-m, spacing-xl, spacing-xs, spacing-xxs

Duplicate colors found: green, purple, teal
Duplicate text styles found: Base Heading, Body, Highlight
Duplicate spacings found: spacing-m, spacing-xl, spacing-xs>
```

I can dive into the tests a bit later but not sure this is expected.